src/http.js

```diff
@@ -255,47 +240,67 @@ const ndjson = async function * (source) {
   }
 }
 
-const streamToAsyncIterator = function (source) {
-  if (isAsyncIterator(source)) {
+/**
+ * Stream to AsyncIterable
+ *
+ * @template TChunk
+ * @param {ReadableStream<TChunk> | NodeReadableStream | null} source
+ * @returns {AsyncIterable<TChunk>}
+ */
+const fromStream = (source) => {
+  if (isAsyncIterable(source)) {
     // Workaround for https://github.com/node-fetch/node-fetch/issues/766
     if (Object.prototype.hasOwnProperty.call(source, 'readable') && Object.prototype.hasOwnProperty.call(source, 'writable')) {
-      const iter = source[Symbol.asyncIterator]()
-
-      const wrapper = {
+      const nodeSource = /** @type {NodeReadableStream} */(source)
```

Comment:
Why is this wrapped in `( )`?

Reply:
> Why is this wrapped in `( )`?

It's a way to do a type conversation and is jsdoc version of

```ts
const nodeSource = <NodeReadableStream>source
```